problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,27 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
+const getCountriesSortedByPopulation =(arr) => {
+    return total 
```

Comment:
This problem is so close!!! Your issue is total on line 11. total hasn't been defined. Instead it should be arr because we want to sort the array that we're getting as the argument.